README.md

```diff
@@ -119,11 +119,11 @@ not receive host hardware information via `/sys/class/hwmon` or `/sys/class/ther
 querying for components may return no results (or unexpected results) when using this library on
 virtual systems.
 
-### Use in binaries distributed on the MacOS App Store
+### Use in binaries running inside the macOS or iOS Sandbox
 
-Apple has restrictions as to which APIs can be used in binaries distributed through the app store.
+Apple has restrictions as to which APIs can be linked into binaries that are distributed through the app store.
 By default, `sysinfo` is not compatible with these restrictions. You can use the `apple-app-store`
-feature flag to disable the Apple prohibited features.
+feature flag to disable the Apple prohibited features. This also enabled the `apple-sandbox` feature. In the case of applications using the sandbox outside of the app store, the `apple-sandbox` feature can be used alone to avoid causing policy violations at runtime.
```

Comment:
```suggestion
feature flag to disable the Apple prohibited features. This also enables the `apple-sandbox` feature.
In the case of applications using the sandbox outside of the app store, the `apple-sandbox` feature
can be used alone to avoid causing policy violations at runtime.
```

Reply:
This flows much better, thank you.